recirq/benchmarks/rabi_oscillations_test.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+from cirq import GridQubit
+from cirq import sim
```

Comment:
usually we try to just `import cirq` in user code (which tests are supposed to imitate) and then use `cirq.Thing`.